content/posts/la-propagacion-de-dns-no-existe.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "La propagación del DNS no existe"
+keywords:
+    - dns
+tags:
+    - dns
+thumbnail: assets/la-propagacion-de-dns-no-existe.png
+socialImage: assets/la-propagacion-de-dns-no-existe.png
+featuredImage: assets/la-propagacion-de-dns-no-existe.png
+date: 2021-04-18T20:32:43-03:00
+draft: true
+---
+
+La propagación del DNS no existe
+
+Una falacia muy extendida entre los profesionales de la informática es que el [DNS](https://es.wikipedia.org/wiki/Sistema_de_nombres_de_dominio) se propaga por alguna red. Tan extendida, de hecho, que hay un par de sitios dedicados a visualizar la propagación geográfica de los registros DNS. Pero la propagación del DNS no existe.
+
+<!--more-->
+
+Un [artículo](https://www.nslookup.io/blog/dns-propagation-does-not-exist/) de [Ruurtjan Pul](https://twitter.com/Ruurtjan)
+
+# Entonces, ¿cómo funciona?
+
+Cuando se solicitan registros DNS al servidor que es autoritativo para ese dominio (es decir, a quien le "pertenece"), éste acompañará esos registros con un TTL (time to live, tiempo de vida). Esto indica cuánto tiempo puede seguir utilizando esos valores antes de que deba solicitar una nueva copia. Los registros se almacenan en una caché en tu dispositivo. Eliminar la necesidad de consultar el servidor DNS cada vez acelera las cosas y descarga los servidores DNS.
+
+Muchos proveedores de servicios de Internet proporcionan servidores de caché DNS a sus abonados. También hay algunos servidores DNS públicos, ofrecidos por [Google](https://developers.google.com/speed/public-dns, [Cloudflare](https://www.cloudflare.com/es-es/learning/dns/what-is-1.1.1.1/) y otros. Todos ellos se denominan servidores DNS recursivos/resolutivos. Pueden proporcionarle una respuesta consultando al servidor DNS autoritativo. Estos servidores recursivos utilizan el TTL para almacenar en caché los registros en su lado. Así que *hay varias capas de caché*: en estos servidores y en tu dispositivo.
```

Comment:
Muchos proveedores de servicios de Internet proporcionan servidores de caché DNS a sus abonados. También hay algunos servidores DNS públicos, ofrecidos por [Google](https://developers.google.com/speed/public-dns), [Cloudflare](https://www.cloudflare.com/es-es/learning/dns/what-is-1.1.1.1/) y otros. Todos ellos se denominan servidores DNS recursivos/resolutivos. Pueden proporcionarle una respuesta consultando al servidor DNS autoritativo. Estos servidores recursivos utilizan el TTL para almacenar en caché los registros en su lado. Así que *hay varias capas de caché*: en estos servidores y en tu dispositivo.